.pre-commit-config.yaml

```
@@ -177,7 +177,7 @@ repos:
     # to XX but somehow the pre-commit hook has version YY when
     # specifying the hook version without the prefix.  See
     # https://github.com/psf/black/issues/2493#issuecomment-1081987650
-    rev: "refs/tags/23.3.0:refs/tags/23.3.0"
+    rev: "23.9.1"
```

```suggestion
    rev: "refs/tags/23.9.1:refs/tags/23.9.1"
```

---

```suggestion
    - repo: https://github.com/psf/black-pre-commit-mirror
      rev: 23.9.1
```

The issue https://github.com/psf/black/issues/2493 seems to be fixed. See https://github.com/psf/black/issues/2493#issuecomment-1664857651.